django/core/models.py

```diff
@@ -556,6 +557,7 @@ def public(self):
 
 @add_to_comses_permission_whitelist
 class Event(index.Indexed, ClusterableModel):
+    EXPIRED_EVENT_DAYS_THRESHOLD = 2
```

Comment:
The constant/setting that does the same thing for jobs (`POST_DATE_DAYS_AGO_THRESHOLD`) is in `settings` (comes from `config.ini`), worth matching where these live? Probably naming too, something like `EXPIRED_JOB_DAYS_THRESHOLD`?